rs/nns/governance/src/governance.rs

```diff
@@ -2895,6 +2895,10 @@ impl Governance {
     /// The source neuron's stake, maturity and age are moved into the target.
     /// Any fees the source neuron are burned before the transfer occurs.
     ///
+    /// TODO: this is not true if the stake is below the transaction fee
```

Comment:
Not clear how to best address this, so if someone from the NNS team has a suggestion, please let me know

Reply:
Not sure what "this" refers to. I guess it refers to "... are moved into the target". Or does "this" refer to "Merge one neuron into another". Not sure what needs to be addressed.